### PR TITLE
chore(Harvest): Update bi-auth to 0.0.8

### PR DESCRIPTION
**Pull request by doubleface**
To get new banking connectors ids into master

A new 2.2.2 version of harvest is already available with this new version of cozy-bi-auth to avoid unneeded feature collisions in home.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -14150,6 +14168,11 @@ react-transition-group@^2.2.1:
     prop-types "^15.6.2"
     react-lifecycles-compat "^3.0.4"
 
+react-zlib-js@^1.0.4:
+  version "1.0.5"
+  resolved "https://registry.yarnpkg.com/react-zlib-js/-/react-zlib-js-1.0.5.tgz#7bb433e1a4ae53a8e6f361b3d36166baf5bbc60f"
+  integrity sha512-TLcPdmqhIl+ylwOwlfm1WUuI7NVvhAv3L74d1AabhjyaAbmLOROTA/Q4EQ/UMCFCOjIkVim9fT3UZOQSFk/mlA==
+
```
Comment (ptbrowne) [inline yarn.lock:14175]: Mmh interesting, why is this added by an update of the bi-auth-package ?

Comment (doubleface) [reply yarn.lock:14175]: This comes from an update of node-jose in bi-auth (even if I do not see any reason to add react in the name)

**Review by ptbrowne — APPROVED** [review]
